midi_app_controller/gui/utils.py

```diff
@@ -149,14 +149,13 @@ def is_subpath(path: Path, subpath: Path) -> bool:
 def reveal_in_explorer(file: Path):
     """Show where the file is in the system's file explorer.
 
-    Currently only Windows and Linux (majority of distributions) are supported.
+    Currently only Windows, Linux (majority of distributions), and macOS are supported.
     """
     if platform.system() == "Windows":
-        subprocess.Popen(rf'explorer /select,"{file}"')
-        return
-
-    if platform.system() == "Linux":
-        subprocess.Popen(rf'xdg-open "{file.parent}"')
-        return
-
-    raise NotImplementedError("Only Linux and Windows are supported")
+        subprocess.Popen([rf'explorer /select,"{file}"'])
+    elif platform.system() == "Linux":
+        subprocess.Popen([rf'xdg-open "{file.parent}"'])
+    elif platform.system() == "Darwin":
+        subprocess.Popen(["open", file.parent])
+    else:
+        raise NotImplementedError("Only Linux and Windows are supported")
```

Comment:
`Only Linux and Windows are supported` -> `Only Linux, Windows, and macOS are supported`

And it still does not work (I only checked Linux). Should be: `subprocess.Popen(["xdg-open", file.parent])`